feature/networkinstance/feature.textproto

```diff
@@ -33,6 +33,17 @@ telemetry_path {
   path: "/network-instances/network-instance/state/type"
 }
 
+# Static routes
+config_path {
+    path: "/network-instances/network-instance/protocols/protocol/static-routes/static/config/prefix"
+}
+config_path {
```

Comment:
We are missing the telemetry state paths for these

Reply:
Done!